fixed typos (#727)
substrate-developer-hub/substrate-docs: v3/tutorials/08-ink-workshop/c-build-erc20-contract/index.mdx

In software engineering practice, it is true that writing automatic test cases cannot be emphasized
enough. There are many type of tests one can write and here we are focusing on writing **Unit Tests**.
This means we as developers know the code logic (versus testing and not knowing the logic, a.k.a.
[black-box testing](https://en.wikipedia.org/wiki/Black-box_testing)) and we write test cases to
verify a function performs as we expected by giving it certain inputs and verifying it is returning
the result we expect. Along the way we also test for edge cases, e.g. how it would handle an empty value,
or a value that is out of its expected bound, and test for certain error handling mechanisms being executed
or error messages being returned.

The benefit of having unit tests written is that once our program gets big or during

> future code refactoring, we can run these tests and if we see them pass, we have the confidence that

Reviewer: ```suggestion
future code refactoring, we can run these tests and if we see them pass, we are confident that
```
